src/content/changes/highlights/2023-11-30.md

```diff
@@ -0,0 +1,12 @@
+---
+date: 2023-11-30T14:00:00
+title: Highlights for the week ending Nov 30 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## General
+
+
```

Comment:
- Logging for workload clusters is now enabled by default
  - You can access those Logs via your installation's Grafana
  - Logs are available for
    - All CAPA workload clusters
    - AWS workload clusters from 19.3.0 onwards
  - Available Logs:
    - Pod Logs from `giantswarm` and `kube-system` namespaces
    - Kubernetes API server audit logs
    - Systemd unit logs
  - Documentation: https://handbook.giantswarm.io/docs/observability/loki-usage/